backend/tracim_backend/lib/core/content.py

```diff
@@ -1598,6 +1598,17 @@ def has_pdf_preview(self, revision_id: int, file_extension: str) -> bool:
             has_pdf_preview = False
         return has_pdf_preview
 
+    def has_jpeg_preview(self, revision_id: int, file_extension: str) -> bool:
+        file_path = self.get_one_revision_filepath(revision_id)
+        try:
+            has_pdf_preview = self.preview_manager.has_jpeg_preview(
```

Comment:
rename `has_pdf_preview` to `has_preview` (or `has_jpeg_preview`). Interest of `has_preview`: being robust to copy/paste

Reply:
done.